utils.py

```diff
@@ -83,6 +192,18 @@ def boxy2mne(mtg_file,tol_file):
         chan_coords.append(source_coords[i_coord] + detect_coords[i_coord])
         chan_labels.append(source_label[i_coord] + '_' + detect_label[i_coord]
                            + ' ' + chan_wavelength[i_coord])
+        
+# =============================================================================
+#     so this has been added because all the .mtg files I have contain
+#     multiple blocks/data files, so a montage file then contains more channels 
+#     than a single boxy data file. this function is only designed to take
```

Comment:
yeah so we will actually want this function to load in all the data files, or we want to call this function repeatedly for each file, but we only need to make the info structure and the locations once

there are two types of files, 001, 002, are different detectors I think (our files now have all detectors in one file). 001a, 001b, 001c, are different montages.

these different montages would have different .mtg and .tol files and so would need unique info for these channels (probably easiest to load a, b, c separately, create info, then merge them?

Reply:
yeah I'm thinking this will be the next big change to work on. Maybe best to make a new branch for this?

Reply:
agreed